lib/jekyll-seo-tag.rb

```diff
@@ -34,7 +27,7 @@ def payload
     def info
       {
         :registers => context.registers,
-        :filters   => [Jekyll::Filters]
+        :filters   => [Jekyll::Filters, Jekyll::Seo::Filters]
```

Comment:
That's fair. We could also add it to Jekyll in the future of that's of interest. People seem to like smart quotes.